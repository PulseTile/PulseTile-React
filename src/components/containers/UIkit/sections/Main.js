import React from 'react';
import ColorPalete from './blocks/ColorPalete';
import Typography from './blocks/Typography';
import Components from './blocks/Components';
import Grid from './blocks/Grid';
import Blocks from './blocks/Blocks';

/**
 * This component returns content of UIkit page
 *
 * @return {XML}
 * @constructor
 */
const Main = () => {
  return (
    <div className="ui-main">
      <div className="ui-content">
        <ColorPalete />
        <Typography />
        <Components />
        <Grid />
        <Blocks />
      </div>
    </div>
  );
};

export default Main;
