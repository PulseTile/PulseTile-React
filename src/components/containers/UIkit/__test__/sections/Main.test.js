import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Main from '../../sections/Main';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Main />', () => {
  it('should renders Main with props correctly', () => {
    const component = shallow(<Main />);
    expect(component).toMatchSnapshot();
  });
});
