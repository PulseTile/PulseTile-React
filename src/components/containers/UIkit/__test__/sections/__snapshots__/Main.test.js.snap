// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Main /> should renders Main with props correctly 1`] = `
<div
  className="ui-main"
>
  <div
    className="ui-content"
  >
    <ColorPalete />
    <Typography />
    <Component />
    <Grid />
    <Blocks />
  </div>
</div>
`;
